docs/management/connectors/action-types/tines.asciidoc

```diff
@@ -0,0 +1,56 @@
+[role="xpack"]
```

Comment:
It seems like the other connectors have a section about how to create a pre-configured one, should we add that here too?